scripts/images/rancher-images.txt

```diff
@@ -16,14 +16,13 @@ docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader:v0.59.
 docker.io/rancher/mirrored-prometheus-operator-prometheus-operator:v0.59.1
 docker.io/rancher/mirrored-prometheus-prometheus:v2.38.0
 docker.io/rancher/mirrored-prometheus-alertmanager:v0.24.0
-docker.io/rancher/rancher-webhook:v0.3.5
-docker.io/rancher/rancher:v2.7.5
-docker.io/rancher/shell:v0.1.8
-docker.io/rancher/shell:v0.1.18
```

Comment:
The job `cattle-monitoring-system/rancher-monitoring-crd-create` needs `docker.io/rancher/shell:v0.1.18`.

Reply:
nice catch! Thanks!